packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx

```diff
@@ -176,19 +196,31 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
           </CallAdapterProvider>
         )}
       </Stack>
-      {(isInLobbyOrConnecting || hasJoinedCall) && (
+      {showControlBar && (
         <ChatAdapterProvider adapter={chatProps.adapter}>
-          <CallWithChatControlBar
-            callAdapter={callAdapter}
-            chatAdapter={chatProps.adapter}
-            chatButtonChecked={showChat}
-            onChatButtonClicked={toggleChat}
-            peopleButtonChecked={showPeople}
-            onPeopleButtonClicked={togglePeople}
-            mobileView={props.formFactor === 'mobile'}
-            disableButtonsForLobbyPage={isInLobbyOrConnecting}
-            callControls={props.callControls}
-          />
+          <Stack.Item styles={controlBarContainerStyles}>
+            <CallWithChatControlBar
+              callAdapter={callAdapter}
+              chatAdapter={chatProps.adapter}
+              chatButtonChecked={showChat}
+              onChatButtonClicked={toggleChat}
+              peopleButtonChecked={showPeople}
+              onPeopleButtonClicked={togglePeople}
+              onMoreButtonClicked={onMoreButtonClicked}
+              mobileView={props.formFactor === 'mobile'}
+              disableButtonsForLobbyPage={isInLobbyOrConnecting}
+              callControls={props.callControls}
+            />
+          </Stack.Item>
+        </ChatAdapterProvider>
+      )}
+      {showControlBar && showDrawer && (
```

Comment:
Why is this piece of code not a part of the `CallWithChatControlBar`?

Reply:
The drawer is a fairly large UI surface that covers the entire screen.

The ControlBar is stacked below the videogallery, whereas this is positioned absolutely on top of the entire stack.